### ISSUE TITLE
Support Network Peering parameters

### ISSUE THREAD

**Issue by fernandocarletti** [pulumi/pulumi-gcp#133]
While setting up Network Peering we ran into some issues with the peering connection configuration. We are not able to enable the "Exchange custom routes" configuration:

![GCP Console](https://i.imgur.com/CYxwS1S.png)

I believe it is the `PeeringNetwork` object in the `addPeering` request: [https://cloud.google.com/compute/docs/reference/rest/v1/networks/addPeering](https://cloud.google.com/compute/docs/reference/rest/v1/networks/addPeering)

**Comment by clstokes** [comment]
@fernandocarletti is this the right resource/setting for you? https://pulumi.io/reference/pkg/nodejs/@pulumi/gcp/compute/#NetworkPeeringArgs-autoCreateRoutes

This is the same as `networkPeering.autoCreateRoutes` at https://cloud.google.com/compute/docs/reference/rest/v1/networks/addPeering.

**Comment by fernandocarletti** [comment]
@clstokes My issue is with `networkPeering.exchangeSubnetRoutes`, which export/import the custom routes in the peering VPC. That's the one I could not find a way to set in Pulumi.

**Comment by clstokes** [comment]
@fernandocarletti The description for `networkPeering.autoCreateRoutes` says `This field will be deprecated soon. Prefer using exchangeSubnetRoutes instead.` so I wonder if it has the same effect?

![image](https://user-images.githubusercontent.com/293763/55765596-40440480-5a25-11e9-88ee-e5c1886cb6e7.png)

(We'll still need to address the deprecation of it.)

**Comment by clstokes** [comment]
@fernandocarletti I think they are functionally equivalent. Can you set `autoCreateRoutes` in Pulumi and see if it works for you?

**Comment by stack72** [comment]
Ping @fernandocarletti just circling back on this one before we either close it out or chase it down

**Comment by fernandocarletti** [comment]
Hello @stack72. I believe in the end this feature wasn't used at Linio (@nesl247 can tell for sure). Seems like the options for configuring the peering in the GCP console are a bit different from the ones available in the API. As far as I remember with the autoCreateRoutes you can't select if you are just importing or just exporting, what was needed in the case I was working on.

**TLDR** I believe we can close this.

**Comment by nesl247** [comment]
We can close this out. If we run into this again we can always reopen it.

**Comment by stack72** [comment]
Thanks to you both :)
